launch-game.sh

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
-# Note: this relies on the non-standard -f flag implemented by gnu readlink
-mono OpenRA.Game.exe Engine.LaunchPath="$(readlink -f $0)" "$@"
-if [ $? != 0 -a $? != 1 ]
+MODLAUNCHER=$(python -c "import os; print(os.path.realpath('$0'))")
+
+# Prompt for a mod to launch if one is not already specified
+MODARG=''
+if [ z"${*#*Game.Mod}" = z"$*" ]
 then
-	ZENITY=`which zenity` || echo "OpenRA needs zenity installed to display a graphical error dialog. See ~/.openra. for log files."
-	$ZENITY --question --title "OpenRA" --text "OpenRA has encountered a fatal error.\nLog Files are available in ~/.openra." --ok-label "Quit" --cancel-label "View FAQ" || xdg-open https://github.com/OpenRA/OpenRA/wiki/FAQ
+	if which zenity
+	then
+		TITLE=$(zenity --forms --add-combo="" --combo-values="Red Alert|Tiberian Dawn|Dune 2000|Tiberian Sun" --text "Select mod" --title="" || echo "cancel")
+		if [ "$TITLE" = "cancel" ]; then exit 0
+		elif [ "$TITLE" = "Tiberian Dawn" ]; then MODARG='Game.Mod=cnc'
+		elif [ "$TITLE" = "Dune 2000" ]; then MODARG='Game.Mod=d2k'
+		elif [ "$TITLE" = "Tiberian Sun" ]; then MODARG='Game.Mod=ts'
```

Comment:
Oh, I thought we were using it for the xdg shortcuts, but it seems the Makefile writes its own copy.